### PR TITLE
Add Divorce transformation endpoint in aat

### PR DESCRIPTION
**Pull request by aliveni**
### Change description ###
Add Divorce transformation endpoint in aat

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/bulk-scan-orchestrator/values.aat.template.yaml [hunk 1]
```diff
@@ -3,3 +3,5 @@ java:
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
   devApplicationInsightsInstrumentKey: "f666440f-43c0-4abb-9c20-2ce175a374f0"
+  environment:
+    TRANSFORMATION_URL_PROBATE: "http://div-cos-aat.service.core-compute-aat.internal"
```
Comment (nitinprabhu) [inline charts/bulk-scan-orchestrator/values.aat.template.yaml]: This is probate transfoturl ?

Comment (aliveni) [reply charts/bulk-scan-orchestrator/values.aat.template.yaml]: no, changed it to divorce

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -81,6 +81,7 @@ service-config:
     - service: divorce
       jurisdiction: DIVORCE
       case-type-ids: []
+      transformation-url: ${TRANSFORMATION_URL_DIVORCE}
```
Comment (nitinprabhu) [inline src/main/resources/application.yaml]: you would need some default value in `values.yaml` else app won't boot up in prod and hat prod containers ?

Comment (aliveni) [reply src/main/resources/application.yaml]: I moved `transformation-url` to `values.yaml` file. I'll raise a pr to update flux repo to set the `transformation-url` value to an empty string in prod.
https://github.com/hmcts/cnp-flux-config/blob/c788e0cdc59d7fb332303a825beb3e6233adea32/k8s/prod/common/bsp/bulk-scan-orchestrator.yaml#L24

Comment (aliveni) [reply src/main/resources/application.yaml]: https://github.com/hmcts/cnp-flux-config/pull/1643

File: charts/bulk-scan-orchestrator/values.yaml [hunk 3]
```diff
@@ -11,6 +11,7 @@ java:
     DOCUMENT_MANAGEMENT_URL: "http://dm-store-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
     TRANSFORMATION_URL_BULKSCAN: "http://bulk-scan-sample-app-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
     TRANSFORMATION_URL_PROBATE: "http://probate-back-office-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+    TRANSFORMATION_URL_DIVORCE: "http://div-cos-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
```
Comment (lgonczar) [inline charts/bulk-scan-orchestrator/values.yaml]: Should this be null by default, so that it's not called in production?

Comment (lgonczar) [reply charts/bulk-scan-orchestrator/values.yaml]: Aaa, read your other email. Ok.

Comment (aliveni) [reply charts/bulk-scan-orchestrator/values.yaml]: yeah, in flux pr I set this value to null in production

**Pull request merged**